### DISCUSSION TITLE
Файлы ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#56] [Announcements]
# Файлы ⚡️ Node.js с примерами кода

Справочник Node.js с примерами кода

https://nodejsdev.ru/guides/webdraftt/files/

**Comment by Bahurtsev** [comment]
Исправьте appendFileAsync() на appendFileSync().

**Reply by bndby** [reply]
Спс, исправил

**Comment by YuraKissil** [comment]
Прошу поправить "одним из двумя способов" -> "одним из двух способов"

**Reply by YuraKissil** [reply]
Прошу поправить "Для записи файла через потока ввода имеется метод" -> "Для записи файла через поток ввода имеется метод"
